htdocs/include/defines.php

```diff
@@ -67,7 +67,7 @@
 define('_XOOPS_FATAL_MESSAGE', "<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>
 <html xmlns='http://www.w3.org/1999/xhtml'>
 <head>
-<meta http-equiv='Content-Type' content='text/html; charset=utf-8;charset=utf-8' />
+<meta http-equiv='Content-Type' content='text/html; charset=utf-8;' />
```

Comment:
Yes I read about it. We could use a cache file for displaying a template file. When the cache was not available, then we would show this ugly define!